cmd/client-s3_test.go

```diff
@@ -219,8 +219,8 @@ func (s *TestSuite) TestObjectOperations(c *C) {
 
 	var reader io.Reader
 	reader = bytes.NewReader(object.data)
-	n, err := s3c.Put(reader, int64(len(object.data)), map[string][]string{
-		"Content-Type": {"application/octet-stream"},
+	n, err := s3c.Put(reader, int64(len(object.data)), map[string]string{
```

Comment:
We should open another bug to introduce context in `mc`

Reply:
not sure how context would be useful since the user interacts with mc only via command line.